twitterpuller/src/main/java/de/brandwatch/minianalytics/twitterpuller/twitter/TweetSearcher.java

```diff
@@ -5,14 +5,11 @@
 import de.brandwatch.minianalytics.twitterpuller.postgres.cache.QueryCache;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.scheduling.annotation.Scheduled;
-import org.springframework.stereotype.Component;
 import twitter4j.*;
 
 import java.time.Instant;
 
-@Component
+//@Component
```

Comment:
Why is this and below commented out?

Reply:
I commented it out to disable the search till we got the unique resource filter merged. Otherwise there would be thousands of duplicate mentions.